src/txkube/testing/test/test_eliot.py

```diff
@@ -33,6 +35,6 @@ def test_tree(self):
         # I don't know exactly what the tree rendering looks like.  That's why
         # I'm using eliot-tree!  So this assertion is sort of lame.
         self.assertThat(
-            _eliottree(events).decode("utf-8"),
+            ensure_unicode(_eliottree(events), encoding="utf-8"),
```

Comment:
`_eliottree` is documented to return `bytes`.  Did you find a case where it returns `unicode`?

Reply:
I removed this in the updated patch.